cgfs.py

```diff
@@ -1258,6 +1258,7 @@ def get_submission(self, path):
         try:
             submission.tld
         except AttributeError:
+            assert submission.children_loaded
```

Comment:
Why assert this here? I thought this should be set in `load_submission_files` which is called *after* this...